src/hosted-form/iframe-content/hosted-input.ts

```diff
@@ -85,6 +88,8 @@ export default class HostedInput {
         this._form.removeEventListener('submit', this._handleSubmit);
         this._unloadFonts();
 
+        window.removeEventListener('focus', () => this.firefoxAndSafariBugfix(this._input));
```

Comment:
I don't think you're removing anything at the moment because you need to pass an actual reference to the callback function that's passed into `addEventListener`. So you'll need to do something like this:

```ts
private firefoxAndSafariBugfix = (targetField: HTMLInputElement) => {
```

And then 
```ts
window.addEventListener('focus', this.firefoxAndSafariBugfix);
```
```ts
window.removeEventListener('focus', this.firefoxAndSafariBugfix);
```

Reply:
I learned the differences between function expression and declaration within a class. Thank you as alaways. 🙇🏻‍♂️